Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json

```diff
@@ -4979,6 +4979,19 @@
   "subtitle": "",
   "provider": "Microsoft"
 },
+{
+    "workbookKey": "ThreatIntelligenceWorkbook",
+    "logoFileName": "",
+    "description": "Gain insights into threat indicators ingestion and search for indicators at scale across Microsoft 1st Party, 3rd Party, On-Premises, Hybrid, and Multi-Cloud Workloads. Indicators Search facilitates a simple interface for finding IP, File, Hash, Sender and more across your data. Seamless pivots to correlate indicators with Microsoft Sentinel: Incidents to make your threat intelligence actionable.",
+    "dataTypesDependencies": [ "ThreatIntelligenceIndicator", "SecurityIncident" ],
+    "dataConnectorsDependencies": [ "ThreatIntelligence", "ThreatIntelligenceTaxii" ],
+    "previewImagesFileNames": [ "ThreatIntelligenceWhite.png", "ThreatIntelligenceBlack.png" ],
+    "version": "1.0.0",
+    "title": "Threat Intelligence",
+    "templateRelativePath": "ThreatIntelligence.json",
```

Comment:
Also please change the "templateRelativePath" and workbook file name as that is also same as [here](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Threat%20Intelligence/Workbooks).

Reply:
Addressed in PR #7776